docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.5'
```

Comment:
Any particular reason you chose 3.5 exactly instead of 3 (wider compatibility)?

Reply:
Right now we're on python 3.5, which was the stable version when development started. When we're ready to do validation work for 3.6 or 3.7 we can bump this up.

Reply:
Yeah, this is the [Compose file version](https://docs.docker.com/compose/compose-file/compose-versioning/). I recommend changing this to `'3'` to allow for wider compatibility, since I don't see any 3.5-specific features in this Compose file.

By the way, if the project _is_ on Python 3.5, [this line](https://github.com/qmk/qmk_api/pull/30/files#diff-3254677a7917c6c01f55212f86c57fbfR1) should be changed so it pulls the correct Python base image.

Reply:
Sorry, bad assumption on my part. :)